packages/block-editor/src/components/use-moving-animation/index.js

```diff
@@ -127,15 +126,15 @@ function useMovingAnimation(
 		reset: triggeredAnimation !== finishedAnimation,
 		config: { mass: 5, tension: 2000, friction: 200 },
 		immediate: prefersReducedMotion,
-		onFrame: ( props ) => {
+		onFrame: () => {
```

Comment:
```suggestion
		onFrame() {
```

Reply:
Do we have a preference? If so, might be nice to have a listing rule. I just left the old code here.

Reply:
Well, the method syntax creates a named function, which is preferable since:

>it helps dramatically in error stack traces, debugger call stacks, and performance audits to quickly identify a function by name (vs. "(anonymous)").

(Quoting @aduth from #22647.)

Also, it's shorter. :stuck_out_tongue:

Whatever the case, I do agree we should probably have a lint rule preferring one or the other. There's a standard ESLint rule that can do this: [`object-shorthand`](https://eslint.org/docs/rules/object-shorthand).

Reply:
In this usage, the name still may be able to be inferred, based on the property that the function is assigned to.

```js
const fns = { a: () => {} }
console.log( fns.a.name );
// "a"
```

I haven't tested if it makes a difference in the debugger.

`object-shorthand` is already in use in the default configuration:

https://github.com/WordPress/gutenberg/blob/de4ea80775e57e088a6fbf389b4ec14d165bc883/packages/eslint-plugin/configs/esnext.js#L24

I wasn't aware, but apparently the usage here is an additional option `avoidExplicitReturnArrows`, which isn't enabled by default:

https://eslint.org/docs/rules/object-shorthand#avoidexplicitreturnarrows

I'm not sure how well it would handle a situation where the semantics of arrow function vs. function expression could have an impact (e.g. `this` binding).

In the end, I'd still consider it a minor detail, and not really one blocking a merge, though it'd be good to have some consistency here in the broader project.